yucca/training/configuration/configure_callbacks.py

```diff
@@ -55,53 +72,77 @@ def get_loggers(
 
     loggers = [
         YuccaLogger(
-            disable_logging=disable_logging,
+            disable_logging=not enable_logging,
```

Comment:
should this be enable_logging=enable_logging now for consistency?

Reply:
Yeah I guess, but we can always change it later. I just noticed that we every else did `if not disable_logging`, so might as well not in once.

Reply:
I think it would be nice to change the variable everywhere if we start calling it something else